float64 frequency  
int8[] datatypes 
float64[] ee_points
float64[] ee_points_tgt
string time_series_topic_name
---
uint64[] time_series_remapping
